tsl/src/bgw_policy/continuous_aggregate_api.c

```diff
@@ -136,6 +136,16 @@ policy_refresh_cagg_get_refresh_end(const Dimension *dim, const Jsonb *config, b
 	return res;
 }
 
+bool
+policy_refresh_cagg_get_include_tiered_data(const Jsonb *config, bool *isnull)
+{
+	bool found;
+	bool res = ts_jsonb_get_bool_field(config, POL_REFRESH_CONF_KEY_INCLUDE_TIERED_DATA, &found);
+
+	*isnull = !found;
+	return res;
```

Comment:
Nit:
```suggestion
	return ts_jsonb_get_bool_field(config, POL_REFRESH_CONF_KEY_INCLUDE_TIERED_DATA, isnull);
```

The code is so simple that I'm not even sure it requires a function, but that's your decision.

Reply:
+1

Reply:
Thanks for the review! Yeah, I would also like to do that, but the `ts_jsonb_get_bool_field` func sets the flag if the key is **found**, not if it is **null**, i.e. the opposite semantics. And I wanted to keep my function's signature in check with the rest of the funcs in this group, i.e. have `isnull` flag.

Reply:
Mea culpa. Please keep the original code then.